src/shared/tpm2-util.c

```diff
@@ -2119,6 +2119,8 @@ int tpm2_create_primary(
                         /* creationData= */ NULL,
                         /* creationHash= */ NULL,
                         /* creationTicket= */ NULL);
+        if (rc == TPM2_RC_BAD_AUTH)
+                return log_debug_errno(SYNTHETIC_ERRNO(EDEADLK), "TPM is locked, cannot generate primary key.");
```

Comment:
and here too, say "Authorization failure while attempting to enroll SRK into TPM"